inspirehep/modules/theme/jinja2filters.py

```diff
@@ -745,3 +748,33 @@ def weblinks(description):
     if description:
         return 'Link to ' + description
     return 'Link to fulltext'
+
+
+@blueprint.app_template_filter()
+def jobs_similar(id):
+
+    out = ''
+
+    es_query = RecordsSearch(index='records-jobs', doc_type='jobs')
+    es_query = es_query.query(
+        {
+            "more_like_this": {
+                "docs": [
+                    {
+                        "_id": id
+                    }
+                ],
+                "min_term_freq": 0,
+                "min_doc_freq": 0,
+            }
+        }
+    )[0:2]
```

Comment:
I don't have the domain knowledge to validate if this choice of a simple MLT without any boosting returns appropriate results. What I propose is that we deploy, and track click to those links: if there are not that many, then we revise the query : )